Makefile

```diff
@@ -0,0 +1,12 @@
+# START: compile
+compile:
+	protoc api/v1/*.proto \
+		--go_out=. \
+		--go-grpc_out=. \
+		--go_opt=paths=source_relative \
+		--go-grpc_opt=paths=source_relative \
+		--proto_path=.

```

Comment:
install the protobuf compiler with go plugin using these steps

Install go library using: `go get -u github.com/golang/protobuf/{proto,protoc-gen-go}`

Run `vim ~/.bash_profile`
Add:
```
export GO_PATH=~/go
export PATH=$PATH:/$GO_PATH/bin
```
Run source `~/.bash_profile`


https://stackoverflow.com/a/57730314